pkgs/development/tools/misc/indent/default.nix

```diff
@@ -10,14 +13,23 @@ stdenv.mkDerivation rec {
   };
 
   patches = [ ./darwin.patch ];
+  postPatch = ''
+    sed -i 's/AC_AIX/AM_PROG_AR/' configure.ac
+  '';
 
-  buildInputs = [ texinfo ];
+  preConfigure = ''
+    sed -i 's/AR = ar/AR = @AR@/' intl/Makefile.in
```

Comment:
Instead of the sed substitutions and autoreconf, we can simply provide `makeFlags = [ "AR=${stdenv.cc.targetPrefix}ar" ];`

Reply:
Thanks. I keep forgetting about cc.targetPrefix and keep reinventing the wheel.